roles/migrationcontroller/tasks/main.yml

```diff
@@ -494,3 +494,5 @@
         phase: Failed
     when: olm_managed and not(reconciled)
 
+- when: migration_controller|bool and noobaa|bool
```

Comment:
I think we want to change this to  `noobaa|bool`. Installing NooBaa probably shouldn't be dependent on installing the controller. I guess it is at least feasible someone would want to put the UI/Controller on OpenShift 3 and NooBaa on OpenShift 4 (or vice versa).